app/assets/javascripts/components/Register.jsx

```diff
@@ -0,0 +1,107 @@
+class Register extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {"success": null};
+  }
+
+  FieldGroup({ id, label, help, ...props }) {
+    const { Checkbox, Radio, FormGroup, ControlLabel, FormControl, Button, HelpBlock } = ReactBootstrap;
+    return (
+      <FormGroup controlId={id}>
+        <ControlLabel>{label}</ControlLabel>
+        <FormControl {...props} />
+        {help && <HelpBlock>{help}</HelpBlock>}
+      </FormGroup>
+    );
+  }
+
+  createUser = (evt) => {
+    evt.preventDefault()
+    const inputs = evt.target.getElementsByTagName("input");
```

Comment:
this isn't that react-y i feel, probably better to use refs or controlled form components